spec/requests/event_streams_spec.rb

```diff
@@ -0,0 +1,137 @@
+RSpec.describe "Event Streams" do
+  describe "GET /api/event_streams" do
+    it "returns a list of event streams with the appropriate role" do
+      api_basic_authorize(action_identifier(:event_streams, :read, :collection_actions, :get))
+      event_stream = FactoryGirl.create(:miq_event)
+
+      get(api_event_streams_url)
```

Comment:
If it a good idea to leave this open to unfiltered requests? If no, it raises the issue - when would we consider a request sufficiently filtered?

Reply:
@imtayadeway does the event streams table grow as rapidly as the metric rollups table? If we were to do filtering, I would suggest we keep it consistent with the metric rollups. Not all of the same filters may apply, but it may be a good idea to require a `start_date` (if interested in the details for metric rollup filtering https://github.com/ManageIQ/manageiq-api/pull/4 )